installer-scripts/SwiftFormat-amd64.wxs

```diff
@@ -0,0 +1,52 @@
+﻿<Wix xmlns="http://wixtoolset.org/schemas/v4/wxs" xmlns:ui="http://wixtoolset.org/schemas/v4/wxs/ui">
+  <Package
+      Language="1033"
+      Manufacturer="nicklockwood"
+      Name="SwiftFormat for Windows x86_64"
+      UpgradeCode="98e01ac8-a17d-43fd-99ed-1cd8b58715bf"
+      Version="$(var.ProductVersion)"
+      Scope="perMachine">
+    <SummaryInformation Description="SwiftFormat for Windows x86_64" />
+
+    <!-- NOTE(compnerd) use pre-3.0 schema for better compatibility. -->
+    <Media Id="1" Cabinet="SwiftFormat.cab" EmbedCab="yes" />
+
+    <!-- Directory Structure -->
+    <!-- WindowsVolume is not a StandardDirectory value, but rather a standard property. See https://github.com/wixtoolset/issues/issues/7314 -->
+    <SetDirectory Id="WINDOWSVOLUME" Value="[WindowsVolume]" />
+    <Directory ComponentGuidGenerationSeed="7818d7fe-5173-4a41-9809-e72263ea9738" Id="WINDOWSVOLUME">
+      <Directory Id="INSTALLDIR">
+        <Directory Id="Library" Name="Library">
+          <Directory Id="Developer" Name="Developer">
+            <Directory Id="Tools" Name="Tools">
+            </Directory>
+          </Directory>
+        </Directory>
+      </Directory>
+    </Directory>
+
+    <!-- Components -->
+    <ComponentGroup Id="SwiftFormat">
+      <Component Id="swiftformat.exe" Directory="Tools" Guid="77126634-5f91-40a7-b344-035ce99ef46f">
```

Comment:
This would be a good opportunity to try removing Component-specific guids. I believe `ComponentGuidGenerationSeed` enables all of these guids to be generated deterministically. Full discussion here:

https://github.com/orgs/wixtoolset/discussions/7337

```suggestion
      <Component Id="swiftformat.exe" Directory="Tools">
```

Reply:
@tristanlabelle I'm getting this error after dropping `Guid=...`

`  D:\a\swift-build\swift-build\SourceCache\installer-scripts\SwiftFormat.wxs(35): error WIX0230: The Component/@Guid attribute's value '*' is not valid for this component because it does not meet the criteria for having an automatically generated guid. Components using a Directory as a KeyPath or containing ODBCDataSource child elements cannot use an automatically generated guid. Make sure your component doesn't have a Directory as the KeyPath and move any ODBCDataSource child elements to components with explicit component guids. [D:\a\swift-build\swift-build\SourceCache\installer-scripts\SwiftFormat.wixproj]`

`Component` indeed has an a `Directory` attribute:

`      <Component Id="swiftformat.exe" Directory="Tools">`

Reply:
I wonder if we should just merge the directory layout and component bits.  That would allow us to remove the `Directory` attribute right?

Reply:
Ah if that syntax is valid then that would make sense! I was wondering what was the alternative to what we are doing.

Reply:
@compnerd Any examples or reference on how to merge directory/component? Apple's wsx files follow this unmerged pattern.

Reply:
@hjyamauchi the wxs files there were authored by me and originally in this repository :)

I think that the `ComponentGroup` will need to be replaced by `Component`s as per the WiX Schema: https://wixtoolset.org/docs/schema/wxs/directory/

Reply:
Ok I think I found a way to avoid guid in the .exe component by removing the component group and moving the component into a directory. But it still complains about the EnvironmentVariables component's missing guid when it's moved into a directory.

Reply:
According to this: "Generatable guids are supported only for components with a single file as the component's keypath or no files and a registry value as the keypath." It sound slike the .exe component can have generated guid but not the environment variable one because the latter doesn't have a file or registery value as a child in this case.

Reply:
https://github.com/compnerd/swift-build/pull/549 is what I got far. @compnerd @tristanlabelle Do you see a way to use autogenerated GUID for the environment variable component?

Reply:
I don't but it's probably no big deal? I think the guids were local to the installer and even if they were not, it might make sense for the guids to clash if you tried to run multiple installers setting the same environment variable. (not sure)